cmd/cape/cmd/token.go

```diff
@@ -67,12 +82,24 @@ func token(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	_, err = cmd.OutOrStdout().Write([]byte(tokenString + "\n"))
-	if err != nil {
-		return err
+	if o != "json" {
```

Comment:
There's no need for this with the renderer. Instead of an empty template, use `{{ .Token }}\n` and the renderer will handle this.